assets/sass/01-settings/file-header.scss

```diff
@@ -15,4 +15,12 @@ Tags: one-column, flexible-header, accessibility-ready, custom-colors, custom-me
 
 Twenty Twenty One WordPress Theme, (C) 2020 WordPress.org
 Twenty Twenty One is distributed under the terms of the GNU GPL.
+
+Twenty Twenty One bundles the following third-party resources:
```

Comment:
I would leave this only in the readme, or the CSS file is going to grow much bigger.

Reply:
Oh cool — I thought the license info needed to be in the stylesheet. I'll move to the readme. 👍

Reply:
Moved over in 72add4d

Reply:
If you scroll down the readme there is already a section for assets:
https://github.com/WordPress/twentytwentyone/commit/72add4d14f2f1d984a85ee14b75e68a88e518379#diff-eb6b6c90251ab33cee784713c451e6d8R70

Reply:
Thanks, I missed that. Updated in c47e22a